classes/State.php

```diff
@@ -232,6 +241,18 @@ public function setInstallVersion($install_version)
         return $this;
     }
 
+    /**
+     * @param string $originVersion
+     *
+     * @return self
+     */
+    public function setOriginVersion($originVersion)
```

Comment:
```suggestion
    public function setOriginVersion(string $originVersion)
```

Reply:
We can't do this kind of changes in the autoupgrade module, it must be compatible with older PS versions :)